en/admin-guide/data-export/interlisexport.rst

```diff
@@ -0,0 +1,180 @@
+.. _export-interlis:
+
+Export INTERLIS Data
+====================
+
+This represents a guide on how to export data from QGEP to INTERLIS 2 Transfer Format (xtf).
+
+
+General
+^^^^^^^^^^^^^
+
+INTERLIS Export can be done using the ili2pg tool from Eisenhut Informatik and a series of prepared SQL scripts. They will do the following:
+* Translation Englisch – German / French (Classes / Attributes)
+* Conversion of value list to / from numeric values (Integer)
+* Restructuring of data from relational to object oriented modelling
+* Conversion of OID (16 characters) to foreign keys TID (integer)
+
+ .. figure:: images/interlisexport.jpg
+ 
+Step by step
+^^^^^^^^^^^^^
+
+Tool
+--------
+Download the latest version of `ili2pg <http://eisenhutinformatik.ch/interlis/ili2pg>`_
+
+ .. figure:: images/eisenhut.jpg
+
+Generate ili2pg schema with data model
+--------------------------------------
+
+You need a valid INTERLIS 2 Transferdataset, the corresponding INTERLIS model
```

Comment:
I don't think you need an xtf to create a schema?

With `--schemaimport` instead of `--import` the .ili can be used in place of a dummy xtf.